frontend/src/notApp.js

```diff
@@ -0,0 +1,60 @@
+import React, { Component } from 'react';
+import Header from './Components/Header/Header';
+import './App.css';
+import CommunityProjects from './Components/CommunityProjects/CommunityProjects';
+import CommunityPosts from './Components/CommunityPosts/CommunityPosts';
+import { projectList } from './projectList';
+import { posts } from './posts';
+
+class App extends Component {
```

Comment:
Why the duplicate App component?

Reply:
it was an over site, should have been deleted.